epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
-				devices, and ecommerce.</p>
+				devices, digital rights management softwares and ecommerce.</p>
 
-			<p>The EAA has been approved on 27 June 2019, should be implemented by EU Member States within 28 June 2022
-				and will enter in force from 28 June 2025.</p>
+			<p>The EAA has been approved on 27 June 2019 and will come into force from 28 June 2025.</p>
 
-			<p>This note aims to demonstrate that the technical requirements of the European Accessibility Act related
-				to ebooks are met by the EPUB® standard.</p>
+			<p>This note aims to demonstrate that EPUB® standard meets all the technical requirements related
+				to ebooks defined by the European Accessibility Act.</p>
 
 			<p>The methodology used is mapping the requirements of the EU Accessibility Act related to ebooks to the <a
 					href="https://www.w3.org/TR/epub-a11y-11/">EPUB Accessibility</a> specification.</p>
 		</section>
-		<section id="sotd"></section>
 		<section id="introduction">
 			<h2>How EPUB conforming to EPUB accessibility guidelines cover European Accessibility Act requirements</h2>
 
-			<p>EPUB is the standard format for the creation and distribution of publications in the digital publishing
-				trade sector.</p>
+			<p>EPUB is the standard format for the creation and distribution of publications in the digital publishing sector.</p>
 
 			<p>The specifications of the EPUB standard are based on the <a
-					href="https://www.w3.org/wiki/Open_Web_Platform">Open Web Platform</a>: HTML, CSS, JavaScript and
-				SVG are the core technologies used for content creation. The use of the latter means that EPUB
-				publications can be written with a high degree of accessibility simply through the proper application of
-				established web accessibility techniques, namely the <a
-					href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines</a>
-				(WCAG).</p>
+					href="https://www.w3.org/wiki/Open_Web_Platform">Open Web Platform</a>: HTML, CSS, XML, MathML and
+				SVG are the core technologies used for content creation. This means that EPUB
+				publications can be created with a high degree of accessibility simply through the proper application of
+				established digital accessibility techniques.</p>
 
-			<p>In order to outline how to make fully accessible EPUBs, the <a href="https://www.w3.org/TR/epub-a11y-11/"
-					>EPUB Accessibility</a> guidelines have been drafted to address two key needs in the EPUB
-				ecosystem:</p>
+			<p>The <a href="https://www.w3.org/TR/epub-a11y-11/">EPUB Accessibility</a> specifications [epub-a11y-11] address the following key areas for making fully accessible EPUB publications:</p>
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
```

Comment:
You're referring to WCAG here, at least by using caps, so there should be a reference.

Reply:
Sorry, what does it mean?